awx/sso/tests/unit/test_fields.py

```diff
@@ -115,6 +115,48 @@ def test_internal_value_invalid(self, data, expected):
         assert e.value.detail == expected
 
 
+class TestSAMLUserFlagsAttrField:
+    @pytest.mark.parametrize(
+        "data",
+        [
+            {},
+            {'is_superuser_attr': 'something'},
+            {'is_superuser_value': 'value'},
+            {'is_superuser_role': 'my_peeps'},
+            {'is_system_auditor_attr': 'something_else'},
+            {'is_system_auditor_value': 'value2'},
+            {'is_system_auditor_role': 'other_peeps'},
+        ],
+    )
+    def test_internal_value_valid(self, data):
+        field = SAMLUserFlagsAttrField()
+        res = field.to_internal_value(data)
+        assert res == data
+
+    @pytest.mark.parametrize(
+        "data, expected",
+        [
+            (
+                {
+                    'junk': 'something',
+                    'is_superuser_value': 'value',
+                    'is_superuser_role': 'my_peeps',
+                    'is_system_auditor_attr': 'else',
+                    'is_system_auditor_value': 'value2',
+                    'is_system_auditor_role': 'other_peeps',
+                },
+                {'junk': ['Invalid field.']},
+            ),
+        ],
```

Comment:
Maybe you meant to add another example later, but `parametrize` with only 1 entry isn't a good general practice.

Are these keys like `is_superuser_value` required? If you have 2 junk keys, should they both show as invalid? It wouldn't be hard to brainstorm a 2nd test case.